CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## v0.28.0 [2021-08-04]
+
+_What's new?_
+
+- New tables added
+  - [aws_emr_cluster_metric_isidle](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_emr_cluster_metric_isidle) ([#570](https://github.com/turbot/steampipe-plugin-aws/pull/570))
+
+_Bug fixes_
+
+- Fixed: Exception handling for organization trail arn in `aws_cloud_trail` ([#578](https://github.com/turbot/steampipe-plugin-aws/pull/578))
+
+- Fixed: Panic run time error in hydrate function getAwsVpcRouteTurbotData for `aws_vpc_route` ([#581](https://github.com/turbot/steampipe-plugin-aws/pull/581))
```

Comment:
```suggestion
- Fixed: Route tables with IPv6 routes no longer cause queries to fail in the `aws_vpc_route` table ([#581](https://github.com/turbot/steampipe-plugin-aws/pull/581))
```

We should add more context here so users understand what cases have been fixed they may have been encountering. 

We should try to stay away from mentioning function names in the release notes as well, as these notes are meant to be more readable, and users can look at the code changes to see what functions have been affected.